lib/lang/isEmpty.js

```diff
@@ -7,23 +7,23 @@ var hasValues = require('has-values');
  * functions, numbers, strings, nulls, objects and arrays.
  *
  * ```js
- * utils.lang.isEmpty('a');
- * //=> true
- *
- * utils.lang.isEmpty('');
+ * isEmpty('a');
  * //=> false
  *
- * utils.lang.isEmpty(1);
+ * isEmpty('');
```

Comment:
you think this is a better format? it's definitely cleaner. I started adding `utils` + category, thinking maybe it would make more sense to users...